Source/ACE.Database/ServerPropertyDatabase.cs

```diff
@@ -0,0 +1,220 @@
+using ACE.Database.Models.Shard;
+using log4net;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Storage;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+
+namespace ACE.Database
+{
+    public class ServerPropertyDatabase
```

Comment:
I don't think this class is needed since we've rolled the config into the shard.

I have a few ideas that we can further discuss in discord.